src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -71,7 +72,15 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
     });
 
     await gitPull(nonNullProp(context, 'repo'));
-    await openYAMLConfigFile(context, swaNode);
+
+    const environmentNode = <EnvironmentTreeItem>(await swaNode.loadAllChildren(context)).find(node => {
```

Comment:
Since this is after creating a SWA, I think it's safe for us to just return the first child node since there shouldn't be any other tree item.

You're not really verifying the type anyway since you're just casting it so I feel like we should be able to just do

```suggestion
    const environmentNode = <EnvironmentTreeItem>(await swaNode.loadAllChildren(context))[0]
```

Reply:
If you want to be extra careful, this is what I seemed to do many many months ago

`const productionEnv: EnvironmentTreeItem | undefined = <EnvironmentTreeItem | undefined>(await swaNode.loadAllChildren(context)).find((ti) => { return ti instanceof EnvironmentTreeItem && ti.label === productionEnvironmentName; });`